.travis.yml

```diff
@@ -16,12 +18,3 @@ env:
   - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=chrome
   - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=chrome ALWAYS_LOCATE=false
   - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=chrome PREFER_CSS=1 SELECTOR_STATS=1
-  # - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=phantomjs
-  # - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=phantomjs ALWAYS_LOCATE=false
-  # - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=phantomjs PREFER_CSS=1 SELECTOR_STATS=1
-  - RAKE_TASK=yard:doctest
```

Comment:
Because it is bad practice to comment out code? :)
We can add an issue to get Phantomjs working.